packages/freezed/lib/src/tools/type.dart

```diff
@@ -73,9 +78,41 @@ String resolveFullTypeStringFrom(
     }
   }
 
+  // The parameter is a Interface with a Type Argument that is not yet generated
+  // In this case analyzer would set its type to InvalidType
+  //
+  // For example for:
+  // List<ToBeGenerated> values,
+  //
+  // it would generate:  List<InvalidType>
+  // instead of          List<dynamic>
+  //
+  // This a regression in analyzer 5.13.0
+  if (type is InterfaceType &&
+      type.typeArguments.contains(InvalidTypeImpl.instance)) {
```

Comment:
That logic needs to be recursive. The type argument could have type arguments too.

We probably should define a separate "typeToString" function which we reuse everywhere